app/[locale]/(private)/page.tsx

```
@@ -1,190 +1,46 @@
+  let leaves;
   try {
     const settingsRes = await getUserSettingsForUser();
+    const leavesRes = await getLeavesForUser();
     visaStartDate = settingsRes.result?.visaStartDate ?? new Date();
+    visaExpiryDate = settingsRes.result?.visaExpiryDate ?? new Date();
+    arrivalDate = settingsRes.result?.arrivalDate ?? new Date();
+    leaves = leavesRes.result;
   } catch (err) {
-    // Log error but continue with default date
-    console.error('Failed to fetch user settings:', err);
     throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)